usr/share/rear/finalize/GNU/Linux/42_migrate_network_configuration_files.sh

```diff
@@ -85,7 +89,7 @@ if test -s $TMP_DIR/mappings/routes ; then
     while read destination gateway device junk ; do
     #   echo "$destination $gateway - $device" >> $TARGET_FS_ROOT/etc/sysconfig/network/routes
         if [[ "$destination" = "default" ]]; then
-            for network_file in $TARGET_FS_ROOT/etc/sysconfig/*/ifcfg-*${device}* $TARGET_FS_ROOT/etc/sysconfig/network ; do
+            for network_file in $TARGET_FS_ROOT/etc/sysconfig/*/ifcfg-*${device}* $TARGET_FS_ROOT/etc/sysconfig/network/routes ; do
```

Comment:
this change could be a backward incompatible change because now
$TARGET_FS_ROOT/etc/sysconfig/network
is no longer used - but perhaps that is needed on older systems?